doc/help/kdb-cache.md

```diff
@@ -0,0 +1,45 @@
+# kdb-cache(1) -- Enable, disable or clear the cache
+
+## SYNOPSIS
+
+`kdb cache {enable,disable,clear}`
```

Comment:
What about a possibility to go back to the default?

Reply:
Good idea, implemented!